src/main/cpp/t02_shoes.cpp

```diff
@@ -28,10 +28,32 @@
 
 #include "t02_shoes.h"
 #include <iostream>
+#include <vector>
+#include <algorithm>
 
 
 using namespace std;
 
 int t02_shoes() {
-    
+	int miley, cyrus;
+	cin >> miley >> cyrus;
+	vector <int> ball(cyrus);
+	for (int i = 0; i < cyrus; i++)
+		cin >> ball[i];
+	sort(ball.begin(), ball.end());
+	int wrecking = 0, force = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'wrecking' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4678797)